src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -129,6 +129,10 @@
             new Object[] {"TwoCommonAttributes", "NHD", 0.3333d},
             new Object[] {"MethodsWithDiffParamTypes", "CCM", 0.0476d},
             new Object[] {"TwoCommonAttributes", "SCOM", 0.0d},
+            new Object[] {"NoMethods", "SCOM", Double.NaN},
+            new Object[] {"OneVoidMethodWithoutParams", "SCOM", 0.0d},
```

Comment:
@rexim ok, value for this class and the other won't be ``Nan``, but why is it ``0.0d``, how did you calculate this?

Reply:
@amihaiemil According to [the definition of SCOM](https://github.com/yegor256/jpeek/blob/fc4437faaff683cfecb073cb7e04f3418cdefca9/src/main/resources/org/jpeek/metrics/SCOM.xsl#L38):

> "0" indicates no cohesion at all (i.e. every method deals with an independent set of attributes)

Which is absolutly true for both of the samples. Both of them have a single method that deals with its own attributes and a sinlge ctor that doesn't deal with any attributes at all. No cohesion whatsoever. You can take those methods and move them to another classes and the ctors won't notice a thing.